README.md

```diff
@@ -143,10 +138,10 @@ Image name structure
 ------------------------
 
 1. Platform name (lowercase) - ruby
-2. Platform version(without dots) - 30
+2. Platform version(without dots) - 33
 3. Base builder image - c9s/rhel8
 
-Examples: `ruby-30-c9s`, `ruby-30`
+Examples: `ruby-33-c9s`, `ruby-33`
```

Comment:
Same comment for 3.0

Reply:
Fixed by https://github.com/sclorg/s2i-ruby-container/pull/580/commits/66bf29c3473b78fdb6734dcb7bb2de57cd22ba87